tests/test_alignment.py

```diff
@@ -71,6 +71,9 @@ def _create_wcs_and_datamodel(fiducial_world, shape, pscale):
 
 
 class WcsInfo:
+    """
```

Comment:
This object (and `DataModel`) isn't added by this PR but I think could be removed with the changes in this PR. I'm not suggesting that we remove these as part of this PR. I opened an issue to track their removal: https://github.com/spacetelescope/stcal/issues/273